qupath-gui-fx/src/main/java/qupath/lib/gui/commands/Commands.java

```diff
@@ -1763,22 +1772,13 @@ public static void promptToSimplifySelectedAnnotations(ImageData<?> imageData, d
 		}
 		
 		long startTime = System.currentTimeMillis();
-		for (var pathObject : pathObjects) {
-			ROI pathROI = pathObject.getROI();
-			if (pathROI instanceof PolygonROI) {
-				PolygonROI polygonROI = (PolygonROI)pathROI;
-				pathROI = ShapeSimplifier.simplifyPolygon(polygonROI, altitudeThreshold);
-			} else {
-				pathROI = ShapeSimplifier.simplifyShape(pathROI, altitudeThreshold);
-			}
-			((PathAnnotationObject)pathObject).setROI(pathROI);
-		}
+		QP.simplifySpecifiedAnnotations(pathObjects, altitudeThreshold);
 		long endTime = System.currentTimeMillis();
 		logger.debug("Shapes simplified in " + (endTime - startTime) + " ms");
 		hierarchy.fireObjectsChangedEvent(hierarchy, pathObjects);
 		imageData.getHistoryWorkflow().addStep(
-				new DefaultScriptableWorkflowStep("Simplify annotations",
-						String.format("simplifyAnnotations(getSelectedObjects(), %f)", altitudeThreshold)
+				new DefaultScriptableWorkflowStep("Simplify selected annotations",
+						String.format("simplifySelectedAnnotations(%f)", altitudeThreshold)
```

Comment:
Use of `%f` isn't ideal, regular string concatenation would be better. If I type 20, I end up with
```groovy
simplifySelectedAnnotations(20.000000)
```